hs_explore/management/commands/fill_go_stop_words.py

```diff
@@ -0,0 +1,100 @@
+"""
+This stores ODM2 and CSDMS keywords used for our LDA model.
+Also, this stors English and customized common hydrological
+stop words used in our LDA model
+"""
+from hs_explore.models import LDAWord
+from django.core.management.base import BaseCommand
+from hs_odm2.models import ODM2Variable
+from hs_csdms.models import CSDMSName
+
+
+def fill_csdms_names():
+    """ Store CSDMS names into LDAWord model
+    """
+    csdms_names = list(CSDMSName.list_all_names())
+    splitted_names = set()
+    for csdms_name in csdms_names:
+        if len(csdms_name) <= 1:
+            continue
+        tokens = csdms_name.split(" ")
+        LDAWord.add_word('CSDMS', 'keep', 'name', csdms_name)
+        splitted_names.update(tokens)
+
+    for splitted_name in splitted_names:
+        if len(splitted_name) <= 1:
+            continue
+        LDAWord.add_word('CSDMS', 'keep', 'name', splitted_name)
+
+
+def fill_odm2():
+    """ Store ODM2 names into LDAWord model
+    """
+    odm2_list = list(ODM2Variable.all())
+    modified_list = []
+    for odm2 in odm2_list:
+        tokens = odm2.split(' - ')
+        new_string = tokens[0]
+        if len(tokens) > 1:
+            new_string = tokens[1] + ' ' + tokens[0]
+        modified_list.append(new_string.lower())
+
+    for odm2_word in modified_list:
+        if len(odm2_word) > 1:
+            LDAWord.add_word('ODM2', 'keep', 'name', odm2_word)
+
+
+def fill_stop_words():
+    """ Store common English and customized stop-words into the LDAWord model
+    """
+    customized_stops = ['max', 'maximum', 'minimum', 'origin', '________', 'center',
```

Comment:
Should these be configurable instead of hardcoded?  I don't know how often you plan to update these.

Reply:
I am not sure how often we would like to update this list. Instead of hardcoding or storing this list into a file, is there any other way to deal with it in our system?

Reply:
If you don't think these will be updated much, if at all, then this is fine.  I'd only consider other solutions if you're going to be tweaking these lists.